### ISSUE TITLE
Linux Krom crash at exit

### ISSUE THREAD

**Issue by KTakahiro1729** [armory3d/armory#758]
The console said the following error code after I closed the armory game player.

```
XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":0"
      after 60 requests (60 known processed) with 0 events remaining.
Received signal 11 <unknown> 000000000000

==== C stack trace ===============================

 [0x7fb34eec1be4]
 [0x7fb34ca3f4b0]
 [0x7fb34e420fe8]
 [0x564a3c31e68c]
 [0x564a3c2debf9]
 [0x7fb34cddb6ba]
 [0x7fb34cb1141d]
[end of stack trace]
```

This was reporduced on a more functional .blend, which worked without a noticeable problem

.blend file and .hx file compressed
[fatalIO.tar.gz](https://github.com/armory3d/armory/files/2169219/fatalIO.tar.gz)

system-info
[system-info.txt](https://github.com/armory3d/armory/files/2169216/system-info.txt)
I updated to the latest armory version.

**Comment by adriansnetlis** [comment]
Could be something to do with your optimus system and xorg. What driver setup do you usr? Bumblebee or iGPU disabled entirely? It could be that the xorg server becomes unavailable to your dGPU before it gets turned off. Anyways, it's all just guesses, didn't research.

**Comment by slapin** [comment]
I see the same without optimus.

**Comment by KTakahiro1729** [comment]
Sorry, but I'm a complete noob on GPU drivers and know nothing about, Bumblebee iGPU settings. 
(trying to figure out right now)

As for now, this is my xorg.conf. Hope this is helpful.

```
Section "ServerLayout"
    Identifier "layout"
    Screen 0 "nvidia"
    Inactive "intel"
EndSection

Section "Device"
    Identifier "intel"
    Driver "modesetting"
    BusID "PCI:0@0:2:0"
    Option "AccelMethod" "None"
EndSection

Section "Screen"
    Identifier "intel"
    Device "intel"
EndSection

Section "Device"
    Identifier "nvidia"
    Driver "nvidia"
    BusID "PCI:1@0:0:0"
    Option "ConstrainCursor" "off"
EndSection

Section "Screen"
    Identifier "nvidia"
    Device "nvidia"
    Option "AllowEmptyInitialConfiguration" "on"
    Option "IgnoreDisplayDevices" "CRT"
EndSection

```

**Comment by RobDangerous** [comment]
Fixed in Kode/Kore@4dc08a5f794ffa789a2543a1fe4d6d61fbfdbc97
